### PR TITLE
[iCubErzelli02] - Arms calibration and strain2 upgrade

### PR DESCRIPTION
**Pull request by gsisinna**

### CODE REVIEW

**Review by pattacini — COMMENTED** [review]
Let me ask @davidetome for reviewing it as well.

File: iCubErzelli02/calibrators/right_arm-calib.xml [hunk 1]
```diff
@@ -34,11 +34,11 @@
 
 <param name="CALIB_ORDER">(0 1 2 3) (4) (5 6 7) (8 9 11 13) (10 12 14 15) </param>
 		<action phase="startup" level="10" type="calibrate">
-		    <param name="target">right_arm-mc_wrapper</param>
+		    <param name="target">right_arm-mc_remapper</param>
 		</action>
 
 		<action phase="interrupt1" level="1" type="park">
-		    <param name="target">right_arm-mc_wrapper</param>
+		    <param name="target">right_arm-mc_remapper</param>
```
[the remark this answers is not in the document]
Comment (pattacini) [reply iCubErzelli02/calibrators/right_arm-calib.xml:41]: So very likely you used a setup that was tested by @Nicogene but then brought back to its previous status immediately after.

We cannot merge this change @gsisinna but only the values of the calibration.
Could you please remove these lines from your update branch?

**Review by pattacini — CHANGES_REQUESTED** [review]
Requested change as per dedicated comment.

File: iCubErzelli02/calibrators/left_arm-calib.xml [hunk 2]
```diff
@@ -34,11 +34,11 @@
 <param name="CALIB_ORDER">(0 1 2 3) (4) (5 6 7) (8 9 11 13) (10 12 14 15) </param>
 
 		<action phase="startup" level="10" type="calibrate">
-		    <param name="target">left_arm-mc_wrapper</param>
+		    <param name="target">left_arm-mc_remapper</param>
 		</action>
 
 		<action phase="interrupt1" level="1" type="park">
-		    <param name="target">left_arm-mc_wrapper</param>
+		    <param name="target">left_arm-mc_remapper</param>
```
Comment (pattacini) [inline iCubErzelli02/calibrators/left_arm-calib.xml:41]: Waiting for @davidetome's review, but the same would apply here.

File: iCubErzelli02/hardware/FT/left_arm-eb1-j0_3-strain.xml [hunk 3]
```diff
@@ -16,16 +16,16 @@
...
-                        <param name="major">            1                       </param>    
+                        <param name="major">            2                       </param>    
                         <param name="minor">            0                       </param>     
                     </group>                    
                     <group name="FIRMWARE">
-                        <param name="major">            0                       </param>    
+                        <param name="major">            2                       </param>    
                         <param name="minor">            0                       </param> 
-                        <param name="build">            0                       </param>
+                        <param name="build">            9                       </param>
```
Comment (pattacini) [inline iCubErzelli02/hardware/FT/left_arm-eb1-j0_3-strain.xml:28]: What is this?
Any comment @davidetome ?

File: iCubErzelli02/hardware/FT/left_arm-eb1-j0_3-strain.xml [hunk 4]
```diff
@@ -39,7 +39,8 @@
 
             <group name="SETTINGS">        
                 <param name="acquisitionRate">          10                      </param>
-                <param name="enabledSensors">           id_l_upper_arm_strain   </param>                
+                <param name="enabledSensors">           id_l_upper_arm_strain   </param>
+				 <param name="temperature-acquisitionRate"> 1000                  </param> <!-- 1 seconds -->
```
Comment (pattacini) [inline iCubErzelli02/hardware/FT/left_arm-eb1-j0_3-strain.xml:43]: Same question here.

File: iCubErzelli02/hardware/FT/left_arm-eb1-j0_3-strain.xml (hunk 3 above)
Comment (gsisinna) [reply iCubErzelli02/hardware/FT/left_arm-eb1-j0_3-strain.xml:28]: An FT45 sensor (strain1) has been replaced with a strain2 sensor. After consulting with @marcoaccame we had to modify the configuration files for the version and protocol.

**Comment by pattacini** [comment]
This PR needs to be carefully checked.
It's also pertaining to `master` when the upgrade involved `devel`.
The change seems to contain modifications introduced by the latest upgrade.

Probably, better off aligning T2T.

**Comment by pattacini** [comment]
After a T2T alignment, we decided to close this PR in favor of an upcoming PR against `devel`.

**Pull request closed without merging**